web/themes/custom/votegov/src/sass/uswds-overrides/usa-nav.scss

```diff
@@ -158,7 +205,11 @@
       @include u-padding-x(2);
       @include u-padding-y(1);
       @include u-display('block');
-      color: $base-white;
+      color: var(--usa-nav--submenu-item-text) !important;
```

Comment:
is the `!important` needed? when i removed it seemed to be ok unless im not following what element this is targeting?

Reply:
If it was working for you I think there was some sort of cacheing issue happening with my machine because it was working fine all day and then stopped working unless !important was included. I'm going to remove it and test again. Thanks for checking